<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vending_machine</class>
 <widget class="QWidget" name="vending_machine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>vending_machine</string>
  </property>
  <widget class="QPushButton" name="push10">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="push50">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push100">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>340</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push500">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>400</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>500</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushCoffee">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>210</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Coffee(100)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushTea">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>260</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Tea(150)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushMilk">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>320</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Milk(200)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Reset">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>510</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
